packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -259,80 +248,6 @@ const Injector = ({ children }: { children: React.ReactNode }): JSX.Element => {
       })
   }, [])
 
-  useEffect(() => {
-    if (library) {
-      async function logGasPrice() {
-        console.log('Gas price:', await library?.getGasPrice())
-      }
-
-      const changeNetwork = async (network: L1Network | L2Network) => {
-        const chainId = network.chainID
-        const hexChainId = hexValue(BigNumber.from(chainId))
-        const networkName = getNetworkName(chainId)
-        const provider = library?.provider
-
-        if (isSwitchChainSupported(provider)) {
-          console.log('Attempting to switch to chain', chainId)
-          try {
-            // @ts-ignore
-            await provider.request({
-              method: 'wallet_switchEthereumChain',
-              params: [
-                {
-                  chainId: hexChainId
-                }
-              ]
-            })
-          } catch (err: any) {
-            if (err.code === 4902) {
-              console.log(
-                `Network ${chainId} not yet added to wallet; adding now:`
-              )
-              // @ts-ignore
-              await provider.request({
-                method: 'wallet_addEthereumChain',
-                params: [
-                  {
-                    chainId: hexChainId,
-                    chainName: networkName,
-                    nativeCurrency: {
-                      name: 'Ether',
-                      symbol: 'ETH',
-                      decimals: 18
-                    },
-                    rpcUrls: [rpcURLs[network.chainID]],
-                    blockExplorerUrls: [getExplorerUrl(network.chainID)]
-                  }
-                ]
-              })
-            } else {
-              throw err
-            }
-          }
-        } else {
-          // if no `wallet_switchEthereumChain` support
-          console.log(
-            'Not sure if current provider supports wallet_switchEthereumChain'
-          )
-          // TODO: show user a nice dialogue box instead of
-          // eslint-disable-next-line no-alert
-          const targetTxName = networksAndSigners.isConnectedToArbitrum
-            ? 'deposit'
-            : 'withdraw'
-
-          alert(
-            `Please connect to ${networkName} to ${targetTxName}; make sure your wallet is connected to ${networkName} when you are signing your ${targetTxName} transaction.`
-          )
-
-          // TODO: reset state so user can attempt to press "Deposit" again
-        }
```

Comment:
This part is removed from the new `switchChain` function.

Is it a conscious choice that we want to change the behaviour to fail the switching silently?